engine/src/mblstore.cpp

```diff
@@ -347,9 +354,22 @@ void MCPurchaseNotifyUpdate(MCPurchase *p_purchase)
 	MCEventQueuePostCustom(t_event);
 }
 
+void MCPurchaseCompleteListUpdate(MCPurchase *p_purchase)
+{
+    if (s_completed_purchases == NULL)
+        /* UNCHECKED */ MCListCreateMutable('\n', s_completed_purchases);
+
+    MCListAppend(s_completed_purchases, p_purchase -> prod_id);
+}
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void MCPurchaseClearList(void)
+{
```

Comment:
I don't think we need this method, since it is not used anywhere. We have MCPurchaseClearPurchaseList() that does the same job.